### PR TITLE
refactor: Improved Docs Page Codebase

### PR DESCRIPTION
**Pull request by adity2911**
## Related Issue

#1517 

## Description

I have made the code easier to read and modular, based on the concept of DRY. This code is renders the exact same webpage as before, while also making it readable.
Lines of code halved from 427 to 261.

## Screenshots

image

### CODE REVIEW

**Comment by adity2911** [comment]
okay. well i thought you gave this issue a go after what you said at our discussion on issue #1517

**Comment by priyankarpal** [comment]
> okay. well i thought you gave this issue a go after what you said at our discussion on issue #1517

you can close the PR & issue & just work on the https://github.com/priyankarpal/ProjectsHut/issues/1519 also, we will change the codebase in docs page

**Comment by adity2911** [comment]
well this is a part of the changes supposed to be made for issue #1519 
If you could merge, It will be good for the 1519 issue.

**Comment by priyankarpal** [comment]
> well this is a part of the changes supposed to be made for issue #1519 If you could merge, It will be good for the 1519 issue.

yeah I know but we will change the docs page UI & code then we need to change just work on that issue

**Comment by adity2911** [comment]
Kindly accept this as a level1 issue only. I have worked twice on the same issue.
It's a humble request.

**Comment by adity2911** [comment]
Plus I was assigned this issue by you only.

File: app/docs/page.tsx [hunk 1]
```diff
@@ -1,66 +1,225 @@
...
+  {
+    icon: <UploadCloud size={18} color="white" />,
+    text: "If you encounter this error while commits",
+    code: "husky - pre-commit hook exited with code 1(error)"
+  },
+  {
+    icon: <UploadCloud size={18} color="white" />,
+    text: "Then run this command",
+    code: "pnpm format"
+  },
```
Comment (priyankarpal) [inline app/docs/page.tsx:132]: remove these lines

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Comment by priyankarpal** [comment]
> Okay. Noted.

will merge soon

**Review by priyankarpal — APPROVED** [review]

**Pull request merged**